### ISSUE TITLE
Feature Request: Dark Theme

### ISSUE THREAD

**Issue by qkqsb** [debrouxl/tilp_and_gfm#59]
Not sure if this issue applies to other desktop environments, but on KDE the icons within the application are impossible to see if a dark theme is being used on the system. Could themes somehow be implemented so that these icons can be more visible?

**Comment by adriweb** [comment]
Are you talking about the icons here? https://github.com/debrouxl/tilp_and_gfm/tree/master/tilp/trunk/icons

If someone has time to create such icons, making a "hardcoded-dark" build with icons replaced by the alternative ones, it seems easy enough :P

**Comment by qkqsb** [comment]
Not quite... See attached of what the icons should look like, vs what they do.
[](url
![tilpDark](https://user-images.githubusercontent.com/90565973/187093853-a7644924-ebcb-4b7c-8c6d-2fff9563a3bf.png)
![tilpLight](https://user-images.githubusercontent.com/90565973/187093854-1180f433-f154-49d1-9efc-e5d7ea6be056.png)
)

**Comment by adriweb** [comment]
Oh that sounds like a GTK issue.
I wonder if the branch with GTK+3 has the same problem. Are you able to build and test?
